runtime/jcl/common/java_dyn_methodhandle.c

```diff
@@ -924,15 +924,10 @@ allocateJ9UTF8(JNIEnv *env, jstring name)
 
 	PORT_ACCESS_FROM_ENV(env);
 
-	utf8 = j9mem_allocate_memory(sizeof(U_16) + length, J9MEM_CATEGORY_VM_JCL);
+	utf8 = j9mem_allocate_memory(sizeof(U_16) + length + 1, J9MEM_CATEGORY_VM_JCL);
```

Comment:
This is an extra byte for `NULL` not required by `J9UTF8`. Can we avoid this waste via a parameter to `copyStringToUTF8Helper` like before? I understand this might complicate the manual versioning inside the method but that should not allow an additional unused byte for each `allocateJ9UTF8` call.

Reply:
Fixed in 4cc4617